app/templates/ember_widgets/documentation.hbs

```diff
@@ -100,6 +100,11 @@
       <td>&#39;&#39;</td>
       <td><p>The path of the default option.</p></td>
     </tr>
+    <tr>
+      <td>showTooltip</td>
+      <td>true</td>
+      <td><p>Whether to show a tooltip when hovering over an option.</p></td>
```

Comment:
@pagael

It's open source so she should be able to see this even if she can't write to the repo.